code/src/index.css

```diff
@@ -5,9 +5,354 @@ body {
+}
+
+h2 {
+  color: white;
+  font-size: 20px;
+}
+
+h3 {
+  color: white;
+  font-size: 16px;
+}
+
+h4 {
+  text-align: center;
+  color: white;
+}
+
+/* THE FOLLOWING CLASSES ARE FROM MOVIELIST.JS */
+.movies {
+  display: flex; /* this pushed all posters in to just one row */
+  flex-wrap: wrap; /* this then made all posters display in rows of two but with an uneven margin to the left */
+  /* justify-content: center; this centered the two posters on the page */
+  justify-content: space-evenly;
+  background-size: cover;
+  column-gap: 0.3rem;
+  row-gap: 1.5rem;
+}
+
+.movie-container {
+  position: relative;
+  width: 50%; /* this moved all posters to the left side, no margin */
+  border: solid grey 2px;
+}
+
+.movie-details {
+  opacity: 0; /* when this is active the title and date do not display */
+  position: absolute;
+  left: 5%;
+  bottom: 5%; /*text appears on top of image*/
+}
+
+.details-img {
+  width: 100%; /* displays all images (2 per row) over the full page */
+  height: auto;
+}
+
+.image-overlay {
+  height: 100%; /* this put the image under the wrong title? */
+  width: 100%;
+  position: absolute; /* this put the posters back in the correct place with their titles */
+  color: transparent;
+}
+
+/* HOVER */
+
+/* ChatGPT on this way of targeting: The > symbol in CSS is known as the child combinator selector, 
+and it selects only the direct child element of a given parent element. In the case of the CSS code you 
+provided, using the > symbol between .movie and .movie-details ensures that only the direct child 
+element with the .movie-details class is targeted, and not any other descendant elements that may 
+also have the .movie-details class. */
+
+.movie:hover > .movie-details { 
+  opacity: 1; 
+  transition: 0.5s ease;
+}
+
+.movie:hover > .image-overlay {
+  background-color: rgba(0, 0, 0, 0.6);
+  transition: 0.5s ease;
+}
+
+/* THE FOLLOWING CLASSES ARE FROM MOVIEDETAILS.JS */
+
+.movie-details-container {
+  width: 100vw;
+  height: 100vh;
+  background-size: cover; /* this altered the background to one big image instead of a small one
+  on constant repeat */
+  display: flex; /* this made the poster (smaller image) large and also screwed with the back button */
+  flex-direction: column; /* this made the background cover in full, the small poster disappeared and the
+  back button is now under the header and stretched from side to side */
+  justify-content: flex-end; /*image now takes up full screen and repeats at bottom */
+  position: relative;
+}
+
+.goback-icon {
+  max-width: 5rem;
+  max-height: 5rem;
+  padding: 2rem;
+  margin-left: 1.5rem;
+  filter: invert(0.6);
+}
+
+.goback-icon:hover {
+  transform:scale(1.5);
+  transition: 0.5s ease;
+  opacity: 1;
+  filter: invert(1);
+}
+
+/* We wanted to show this text when hovering over the back-to-list-button but we didn't get it to work
+We will ask this on Mondays team session  */
+/* .back-to-list {
+  display: none;
+} 
+
+.back-to-list:hover {
```

Comment:
The problem might be, that you've put a hover on something that's been set to "display: none" - you can't hover over something that's not there. Have the text + button in a container instead, and add the hover to the container.